moonraker/components/mqtt.py

```diff
@@ -191,6 +191,7 @@ def __init__(self, config: ConfigHelper) -> None:
         self.api_request_topic = f"{self.instance_name}/moonraker/api/request"
         self.api_resp_topic = f"{self.instance_name}/moonraker/api/response"
         self.klipper_status_topic = f"{self.instance_name}/klipper/status"
+        self.klipper_state_topic = f"{self.instance_name}/klipper/state"
```

Comment:
Come to think of it, I do wonder if this should be `{self.instance_name}/moonraker/state` or even just `{self.instance_name}/state`